docs/source/relayer.rst

```diff
@@ -0,0 +1,105 @@
+Cross-chain message relayer
+===========================
+
+| This component is responsible for handling the cross-chain contract state updates. 
+|
+| It is mainly used by the Agent software when resolving requests as a result of a running challenge game
+  or for sending fill invalidation messages (L1 resolutions). However, it can also be used manually by running
+  the CLI commands.
+| If you haven't read about the role of :ref:`l1-resolutions` in Beamer, make sure you get familiar 
+| with this concept first.
+|
+| Basically, every **request fill** action creates a **proof** on the request destination chain.
+  The agent utilizes these proofs in an optimistic way. If a challenge game ever occurs, or if a
+  fill was invalid, the agent has the chance to prove that a misbehavior occured and in the case of a challenge game,
+  get rewarded for it.  
+|
+| Since these **proofs** lay on a different chain than the chain where the requests originated from and where the challenge
+  game is running, we need a way to transport this **proof** from one chain to another in order to be able 
+  to prove that a certain state is correct. Thats where the cross-chain relayer comes into play.
+|
+| For instructions on how to setup, compile and run the relayer, make sure you take a look at the `/relayer/README.md` file.
+
+Command reference
+-----------------
+
+The relayer CLI currently supports the following commands:
+
+
+* :ref:`command-relay` relays a fill proof or a fill invalidation message from chain A to chain B.
+* :ref:`command-prove-op-message` proves on L1 that a message exists on L2. Used only for messages traveling from chains that are based on the *OP Bedrock* stack.
```

Comment:
Nitpick: would it be possible to break this long line somehow?

Reply:
Uhm, not sure. I will give it a try.

Reply:
I'm fine with whatever you choose here, it's just that it drew my attention due to line length, but in the end it's only a nitpick.